app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
+  ManageIQ::Providers::BaseManager::EventCatcher::Runner
+  include ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcherMixin
```

Comment:
Go ahead and just dump the mixin contents in this file.  We really only used the mixins for the providers that had more than one manager type and needed to share the same `EventCatcher::Runner` code.